gradle-plugin/build.gradle

```diff
@@ -28,7 +28,7 @@ group = 'com.google.cloud.tools'
 sourceCompatibility = 1.8
 
 dependencies {
-  implementation 'com.google.cloud.tools:dependencies:1.4.3'
+  implementation 'com.google.cloud.tools:dependencies:latest.integration'
```

Comment:
can we use a specific version  here?

Reply:
This points to the latest master until we release next version. I created a note for next release of the enforcer rule. https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1557

Reply:
Can we release the next version now/first? It does seem weird that we can't point to the current version, probably because we're mixing a maven and gradle build in this repo.

Reply:
No, we cannot. We need to merge this PR's changes in dependencies module first before updating this line to new version.

What needs to happen:
- merge this PR to modify dependencies module so that the classes are visible for the Gradle plugin,
- release the dependencies module.
- update this line to next version.